images/win/Windows2016-Azure.json

```diff
@@ -875,8 +876,11 @@
         },
         {
             "type": "powershell",
+            "environment_vars": [
```

Comment:
Packer can't correctly validate this template `879:  "environment_vars":�`.  Since you put the [Non-breaking_space](https://en.wikipedia.org/wiki/Non-breaking_space ) with code 160, it should be the classic space with code 32.

Reply:
fixed issue with space in my #bd80f20 commit